### PR TITLE
[TS-317] APIs for follow/unfollow a public account

### PR DESCRIPTION
**Pull request by kennguyen0303**
Closes TS-317.

1. Added API for: follow, unfollow, get followers and followings for an account.
2. Edited some comments in method returns
3. Minor debug in test and `unfollow(..)`
4. API documentation is available at: https://documenter.getpostman.com/view/10545271/UVe9QU3y 

Note: For testing on Postman, you will need to first Login, then copy the `accessToken` to `Authentication` field -> Choose `Bearer-token` and paste the token there. 

image

### CODE REVIEW

**Review by bowe99 — APPROVED** [review]
Looks great

File: src/routes/v1/userInfo.route.ts [hunk 1]
```diff
@@ -5,18 +5,21 @@ const userInfoRoute = (app: Express) => {
 	const userInfoController = new UserInfoController();
 	const router = express.Router();
 
+	router.get("/", (req, res) => {
```
Comment (kennguyen0303) [inline src/routes/v1/userInfo.route.ts:8]: @bowe99 the meat :cut_of_meat:

**Review by kennguyen0303 — COMMENTED** [review]
image

@bowe99 api for search by username: `/api/v1/userInfo?searchQuery=<keyword>`

File: src/index.ts [hunk 2]
```diff
@@ -31,12 +31,13 @@ const cleanUpServer = async () => {
 		else {
 			await neo4j.disconnect();
 			await mongoInstance.disconnect();
+			console.log("exit ?");
```
Comment (itsShaVawn) [inline src/index.ts:34]: can you just remove this log?

**Review by itsShaVawn — APPROVED** [review]

**Review by itsShaVawn — APPROVED** [review]
looks good!

**Review by tungleu — APPROVED** [review]

**Pull request merged**